client/src/Components/Pages/Questionnaire/PersonalDetails1/index.js

```diff
@@ -1,65 +1,158 @@
+            
+            }else{
+            this.setState({files:files,label:files[0].name})
+        }
+    }
       
     render() {
-        const { selectedOption } = this.state; 
         return ( 
-            <div>
-                <div className="third-section">
-                    <h2>Now we’ve got the boring stuff out of the way, let’s get to know you better</h2>
-                    <div className="six-card">
-                        <p>What subjects are you studying? Tick all that apply</p>
-                        <Select
-                            value={selectedOption}
-                            isMulti={this.handleChange}
-                            options={options}
-                        />
-                        <p>What are your favourite subjects Azara ? Tick up to 3</p>
-                        <input type="text" className ="sec1-input"/>
-                        <p>What else do you enjoy at school? Tell us as much as you can.</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                        <p>What are your interests and hobbies outside of school? Tell us everything you enjoy doing.</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                    </div>
-                    <div className="seventh-card">
-                        <p>If you have a copy of your CV, please upload it to your dashboard</p>
-                        <input type="file"/>
-                        <p>Can you imagine your future career? What does it look like?</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                        <p>What kinds of jobs do you think will interest you?</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
+            <div className="container">
+                <div>
+                    <h2>Now we've got the boring stuff out of the way, let's get to know more about your school studies</h2>
+                    <div className="third-section">  
+                        <div className="six-card">
+                            <span className="num" id="num6">6</span>
+                            <p>What subjects are you studying? Tick all that apply</p>
+                            <Select
+                                className="select-input"
+                                onChange={this.handleSubjectChange}
+                                value={this.subjects}
+                                isMulti={true}
+                                options={options}
+                            />
+                            <p>What are your favourite subjects Azara ? Tick up to 3</p>
+                            <Select 
+                                className="select-input"
+                                onChange={this.handleFavChange}
+                                value={this.favSubjects}
```

Comment:
This is another spot where you need to use data that's in this.state (so `value={this.state.favSubjects}`)